src/ansys/dpf/core/workflow.py

```diff
@@ -806,6 +809,53 @@ def create_on_other_server(self, *args, **kwargs):
                 "or both ip and port inputs) or a server is required"
             )
 
+    def view(
+            self,
+            viewer: Union[None, str] = None,
+            title: Union[None, str] = None
+    ):
+        """Run a viewer to show a rendering of the workflow.
+
+        Parameters
+        ----------
+        viewer:
+            Available viewers are None and "paraview".
+            If None, the system's default image viewer is used.
+
+        title:
+            Name to use in intermediate files and in the viewer.
+        """
+        if title is None:
+            name = f"workflow_{repr(self).split()[-1][:-1]}"
+        else:
+            name = title
+        file_path = os.path.join(os.getcwd(), f"{name}.dot")
```

Comment:
Do we need a file? Maybe we could add an api similar to `self._api.work_flow_export_graphviz` that generates a big string (the contents of the dot file) that are then passed to graphviz without files

Reply:
@rafacanton I agree, we would need an operator which just returns the contents of the DOT file. But that would mean a new operator, and managing how to feed this big string to the renderer, which currently only accepts files, so we would have to mimick a file... 
I think this is for a future PR when we see the need.
Right now I improved the behavior so we clean-up the DOT file by default.